Source/WebKit/WebProcess/Extensions/WebExtensionContextProxy.h

```diff
@@ -92,6 +92,7 @@ class WebExtensionContextProxy final : public RefCounted<WebExtensionContextProx
     WebCore::DOMWrapperWorld& toDOMWrapperWorld(WebExtensionContentWorldType) const;
 
     static WebCore::DOMWrapperWorld& mainWorld() { return WebCore::mainThreadNormalWorld(); }
+    static Ref<WebCore::DOMWrapperWorld> protectedMainWorld() { return mainWorld(); }
```

Comment:
We should add `Singleton()` to the name instead. `WebCore::mainThreadNormalWorld()` returns a singleton.